### PR TITLE
Match way higher, speed crazy fast

### PR DESCRIPTION
**Pull request by armike**
New awesome matching algorithm

### CODE REVIEW

File: match.py [hunk 1]
```diff
@@ -6,11 +6,13 @@
...
     MATCH_QUALITY_HIGH = 100
+    MATCH_QUALITY_WAY_HIGHER = 1000
     MATCH_SPEED_FAST = 10
+    MATCH_SPEED_CRAZY_FAST = 200
 
     def match_drivers_and_pa(self, passengers, drivers):
-        match_quality = Match.MATCH_QUALITY_HIGH
-        match_speed =  Match.MATCH_SPEED_FAST
+        match_quality = Match.MATCH_QUALITY_WAY_HIGHER
+        match_speed =  Match.MATCH_SPEED_CRAZY_FAST
```
Comment (knowledgeabledev) [inline match.py]: Crazy fast is too fast -- the world is not ready.

File: match.py [hunk 2]
```diff
@@ -6,11 +6,13 @@
...
     MATCH_QUALITY_HIGH = 100
+    MATCH_QUALITY_WAY_HIGHER = 1000
     MATCH_SPEED_FAST = 10
+    MATCH_SPEED_REASONABLY_FAST = 199
 
     def match_drivers_and_pa(self, passengers, drivers):
-        match_quality = Match.MATCH_QUALITY_HIGH
-        match_speed =  Match.MATCH_SPEED_FAST
+        match_quality = Match.MATCH_QUALITY_WAY_HIGHER
+        match_speed =  Match.MATCH_SPEED_REASONABLY_FAST
```
Comment (armike) [inline match.py]: @knowledgeabledev how about now?

Comment (knowledgeabledev) [inline match.py]: @armike it's just crazy enough to work. :+1:

File: match.py [hunk 3]
```diff
@@ -6,12 +6,15 @@
...
     MATCH_SPEED_FAST = 10
+    MATCH_SPEED_CRAZY_FAST = 200
 
-    def match_drivers_and_pa(self, passengers, drivers):
-        match_quality = Match.MATCH_QUALITY_HIGH
-        match_speed =  Match.MATCH_SPEED_FAST
-        self.match_drivers_with_params
+    def match(self, passengers, drivers):
+        match_quality = Matcher.MATCH_QUALITY_WAY_HIGHER
+        match_speed =  Matcher.MATCH_SPEED_CRAZY_FAST
```
Comment (knowledgeabledev) [inline match.py]: Crazy fast is too fast -- the world is not ready.

**Pull request closed without merging**